generator/internal/descriptor_utils.cc

```diff
@@ -641,12 +641,18 @@ std::vector<std::unique_ptr<GeneratorInterface>> MakeGenerators(
     std::vector<std::pair<std::string, std::string>> const& vars) {
   std::vector<std::unique_ptr<GeneratorInterface>> code_generators;
   VarsDictionary service_vars = CreateServiceVars(*service, vars);
+  bool omit_client = false;
+  for (auto const& v : vars) {
```

Comment:
Should you look in `service_vars` instead (and then use `find()`)?

Reply:
Done, for whatever reason my previous attempt at this failed.